### PR TITLE
Remove Armada UUIDs from Event Ingester and Event Api

### PR DESCRIPTION
**Pull request by d80tb7**
Phase two of removing the proto uuids and replacing them with the string versions

### CODE REVIEW

File: internal/server/event/conversion/conversions.go [hunk 1]
```diff
@@ -494,39 +438,12 @@ func FromInternalJobRunPreempted(queueName string, jobSetName string, time time.
...
-	jobId, err := armadaevents.UlidStringFromProtoUuid(e.PreemptedJobId)
-	if err != nil {
-		return nil, err
-	}
-	runId, err := armadaevents.UuidStringFromProtoUuid(e.PreemptedRunId)
-	if err != nil {
-		return nil, err
-	}
-
-	preemptiveJobId := ""
```
Comment (d80tb7) [inline internal/server/event/conversion/conversions.go:506]: we don't set these  anywhere.  We should remove these from the event but we can do that in a separate PR

**Review by JamesMurkin — APPROVED** [review]
